projects/ng-components/lib/radio-group/radio-group.component.ts

```diff
@@ -104,8 +104,10 @@ export class RadioGroupComponent implements ControlValueAccessor, AfterViewCheck
         if (!this.didFocus && this.list && this.list.length) {
             const currentFocused: number = this.list.findIndex(e => e && this.selectedValue && e.value === this.selectedValue.value);
             if (currentFocused > -1 && this.radioRefs.toArray()[currentFocused] && this.radioRefs.toArray()[currentFocused].nativeElement) {
-                this.radioRefs.toArray()[currentFocused].nativeElement.focus();
-                this.didFocus = true;
+                setTimeout(() => {
+                    this.radioRefs.toArray()[currentFocused].nativeElement?.focus();
+                    this.didFocus = true;
+                }, 0);
```

Comment:
maybe we can use changesDetectionRef here instead of using setTimeout

Reply:
I tried but changing the change detection strategy does not fix the scroll jumping issue ( re #443 ). SetTimeout is the only thing that fixes is unfortunately.